controller/controller_server/utils.py

```diff
@@ -201,6 +236,35 @@ def generate_csi_create_snapshot_response(new_snapshot, source_volume_id):
     return res
 
 
+def _get_supported_capabilities(volume_capabilities):
+    if volume_capabilities.HasField(config.VOLUME_CAPABILITIES_FIELD_ACCESS_TYPE_MOUNT):
+        return csi_pb2.VolumeCapability(
+            mount=csi_pb2.VolumeCapability.MountVolume(fs_type=volume_capabilities.mount.fs_type),
+            access_mode=csi_pb2.VolumeCapability.AccessMode(mode=volume_capabilities.access_mode.mode))
```

Comment:
this line is duplicated, no need to make the `AccessMode` instance in more than 1 place

Reply:
done